dev/js/_functions.js

```diff
@@ -376,7 +376,7 @@ function saveFile(fname, buffer, ftype) {
 	function isval(val){
 		if(val === 0) return true;
 		else if (val === false) return true;
-		else if (JSON.stringify(val) === '{}') return false;
+		else if ( typeof val === 'object' && Object.keys(val).length === 0 ) return false;
```

Comment:
If I'm correct, this line was meant to return false when val was just en empty object. In this case JSON.stringify is an inefficient and flawed way to handle it. I had to replace this line to fix import from Prototypo.